### PR TITLE
Bug 1764573 - Only report the number of overflowing tasks in preinit_tasks_overflow metric

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -260,7 +260,7 @@ jobs:
       - run:
           name: Check vendored schema for upstream updates
           command: |
-            bin/update-schema.sh HEAD
+            bin/update-schema.sh main
```
Comment (chutten) [inline .circleci/config.yml:263]: What's this about?

**Review by chutten — APPROVED** [review]

**Comment by chutten** [comment]
Also it looks like you missed another `110` in GleanTest.kt: https://searchfox.org/glean/source/glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt#621

File: .circleci/config.yml (hunk 1 above)
Comment (badboy) [reply .circleci/config.yml:263]: should have left a comment: GitHub changed _something_ and now `HEAD` is not valid anymore in direct URLs to files: https://app.circleci.com/pipelines/github/mozilla/glean/9214/workflows/436eec8e-cab6-405e-b2f6-f1739852ed03/jobs/170397

Comment (Dexterp37) [reply .circleci/config.yml:263]: Yeah, looks like it was a bug that's being fixed :)

**Pull request merged**